PlatformIO ESP32 code/OSSM_ESP32/platformio.ini

```diff
@@ -17,3 +17,6 @@ upload_speed = 921600
 monitor_speed = 115200
 monitor_flags =
             --rtscts
+
+lib_deps =
+        StrokeEngine=https://github.com/theelims/StrokeEngine.git
```

Comment:
to ensure breaking changes of the stroke engine API do not break the build of this project, you may want to pin the version of the library that platformio fetches.

`StrokeEngine=https://github.com/theelims/StrokeEngine.git#v0.3.0`